app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -85,11 +84,17 @@ class AdministratorControlsActivity :
       supportFragmentManager.findFragmentById(
         R.id.administrator_controls_fragment_multipane_placeholder
       )
-
-    if (fragment is ProfileEditFragment)
+    /**
+     * when fragment is [ProfileEditFragment] then we should switch
+     * fragment to [ProfileListFragment] or else
+     * we just end the activity as then we are
+     * having [ProfileListFragment] on top.
+     */
```

Comment:
This should either be a block comment (e.g. ``/* */``) or single-line comments (e.g. ``/``), not a KDoc.

Reply:
Also, we can't use ``[]`` for line comments (they're only valid for KDocs).